cmd/bridged/main.go

```diff
@@ -55,15 +295,47 @@ func main() {
 		&cmd.BlockchainInfo.NetworkName,
 		"network", "n",
 		cmd.BlockchainInfo.NetworkName,
-		"the name of the network to which the daemon connects, one of {standard,testnet,devnet}",
+		"the name of the tfchain network to  connect to  {standard,testnet,devnet}",
 	)
-	NetAddressArrayFlagVar(
+
+	cli.NetAddressArrayFlagVar(
 		cmdRoot.Flags(),
 		&cmd.BootstrapPeers,
-		"bootstrap-peer",
-		"overwrite the bootstrap peers to use, instead of using the default bootstrap peer (requird for devnet)",
+		"bootstrap-peers",
+		"overwrite the bootstrap peers to use, instead of using the default bootstrap peers",
+	)
+
+	// bridge flags
+
+	cmdRoot.Flags().StringVar(
+		&cmd.EthNetworkName,
+		"ethnetwork", "main",
```

Comment:
I agree with forcing networks for mainnet and testnet, though I'd actually prefer to deploy testnet on ropsten, since I feel that would cause the block delay tohave more testing

Reply:
> I agree with forcing networks for mainnet and testnet

we don't force though, that is just about the defaults.

I agree with ropsten for testnet. So perhaps the defaults could be

| TFT | ETH|
|--|--|
| standard | main |
| testnet | ropsten |
| devnet | rinkeby |

But again it's not about forcing, but use those as defaults, in case the EthNetworkName is equal to the empty string (as should be the default). Should the user have entered one of the accepted networks (main/ropsten/rinkeby) by entering it as the EthNetworkName, we should respect it.

Reply:
Forcing might be a bit strong wording, though it would not make sense for a user to link another network for standard/testnet right? so not sure why we would allow that

Reply:
Who knows, perhaps we want to verify some new feature of the bridge against transactions of main net but do so on a newly deployed contract on ropsten, as to be able to test this feature without playing with real money on Ethereum. Dunno, I'm sure there are use cases. The defaults are probably fine for most use cases, but the option to choose is there should one want.